web-react-ts/src/permission-utils.ts

```diff
@@ -164,7 +164,11 @@ export const permitArrivals = (churchLevel: ChurchLevel): Role[] => {
       break
   }
 
-  return [...permitAdmin(churchLevel), ...permittedFor]
+  if (churchLevel !== 'Oversight') {
```

Comment:
I feel you could rather use when the churchLevel is Stream. Because all others should not return arrivals helpers